ubiquerg/system.py

```diff
@@ -25,9 +26,21 @@ def is_command_callable(cmd):
         raise ValueError("Empty command to check for callability")
     if os.path.isdir(cmd) or (os.path.isfile(cmd) and not os.access(cmd, os.X_OK)):
         return False
-    # Use `command` to see if command is callable, and rule on exit code.
-    check = "command -v {0} >/dev/null 2>&1 || {{ exit 1; }}".format(cmd)
-    return not bool(os.system(check))
+    if os.name == "nt":
```

Comment:
can you add a comment here to explain what you're doing?

Reply:
it is just linting